akka-actor/src/main/scala/akka/actor/Address.scala

```diff
@@ -204,7 +205,7 @@ object ActorPathExtractor extends PathUtils {
       val uri = new URI(addr)
       uri.getRawPath match {
         case null => None
-        case path => AddressFromURIString.unapply(uri).map((_, split(path, uri.getRawFragment).drop(1)))
```

Comment:
It seems that the reason we had `.drop(1)` here was to clean-up a trailing whitespace we were leaving in the `split` method.